forge-gui/res/cardsfolder/upcoming/consuming_corruption.txt

```diff
@@ -0,0 +1,7 @@
+Name:Consuming Corruption
+ManaCost:B B
+Types:Instant
+A:SP$ DealDamage | ValidTgts$ Creature,Planeswalker | TgtPrompt$ Select target creature or planeswalker | NumDmg$ X | SubAbility$ DBGainLife | SpellDescription$ CARDNAME deals X damage to target creature or planeswalker and you gain X life, where X is the number of Swamps you control.
+SVar:DBGainLife:DB$ GainLife | LifeAmount$ X
+SVar:X:Count$TypeYouCtrl.Swamp

```

Comment:
Should this be `SVar:X:Count$Valid Swamp.YouCtrl`?

Tendrils of Corruption also uses this syntax

Probably both are ok?

Reply:
yea a bit older variant
maybe one day we remove it?